README.md

```diff
@@ -52,28 +52,36 @@ Main developer:
 
 Contributors (in alphabetical order):
 
-- Antoine Avrit
 - Amishga Alphonius
+- Antoine Avrit
 - Lucka Barbeau
 - Valérie Bibeau
+- Bianca Bugeaud
 - Audrey Collard-Daigneault
 - Carole-Anne Daunais
 - Toni El Geitani
+- Olivier Gaboriault
 - Simon Gauvin
 - Shahab Golshan
+- Olivier Guévremont
 - Marion Hanne
 - Jeanne Joachim
 - Rajeshwari Kamble
 - Martin Kronbichler
+- Maxime Landry
+- Pierre Laurentin
 - Charles Le Pailleur
+- Oreste Marquis
 - Ghazaleh Mirakhori
+- Thomas Monatte
 - Peter Munch
 - Victor Oliveira Ferreira
 - Hélène Papillon-Laroche
 - Paul Patience
 - Catherine Radburn
 - Philippe Rivest
 - Laura Prieto Saavedra
+- Mikael Vaillant
```

Comment:
Can we move my name after Paul? Since my first last name is Prieto...
```suggestion
- Paul Patience
- Laura Prieto Saavedra
- Catherine Radburn
- Philippe Rivest
- Mikael Vaillant
```